src/modules/sensors/sensors.cpp

```diff
@@ -178,10 +185,19 @@ class Sensors : public ModuleBase<Sensors>
 	orb_advert_t	_diff_pres_pub{nullptr};			/**< differential_pressure */
 	orb_advert_t	_sensor_preflight{nullptr};		/**< sensor preflight topic */
 
+	orb_advert_t 	_wind_est_pub{nullptr};			/**< wind estimate topic */
+
 	perf_counter_t	_loop_perf;			/**< loop performance counter */
 
 	DataValidator	_airspeed_validator;		/**< data validator to monitor airspeed */
 
+	differential_pressure_s _diff_pres{};
```

Comment:
All of these together with `WindEstimator` add several 100 bytes to the RAM usage, for something that we don't need in MC (or will we?). Something optional & dynamically allocated might make more sense here.

Reply:
wind data can be very useful for certain types of sensor readings such as gas sensors